File: js/part_1/2.js
let nota = parseFloat(prompt("Introduce tu nota:"));

if (nota >= 0 && nota < 3) {
    console.log("Muy deficiente");
} else if (nota >= 3 && nota < 5) {
    console.log("Insuficiente");
} else if (nota >= 5 && nota < 6) {
    console.log("Suficiente");
} else if (nota >= 6 && nota < 7) {
    console.log("Bien");
} else if (nota >= 7 && nota < 9) {
    console.log("Notable");
} else if (nota >= 9 && nota <= 10) {
    console.log("Sobresaliente");
} else {
    console.log("Nota fuera de rango");
}

File: py/part_1/2.py
nota = float(input("Introduce tu nota: "))

if 0 <= nota < 3:
    print("Muy deficiente")
elif 3 <= nota < 5:
    print("Insuficiente")
elif 5 <= nota < 6:
    print("Suficiente")
elif 6 <= nota < 7:
    print("Bien")
elif 7 <= nota < 9:
    print("Notable")
elif 9 <= nota <= 10:
    print("Sobresaliente")
else:
    print("Nota fuera de rango")
